Makefile

```diff
@@ -134,7 +117,18 @@ endif
 # Whether to colorize build messages
 COLOR ?= 1
 
-
+ifeq ($(OS),Windows_NT)
+    DETECTED_OS=windows
+	# define TMPDIR
+	export TMPDIR=$(TEMP)
```

Comment:
TEMP does not appear to be set anywhere, is it still needed?

Reply:
or it's IDO who need this environment variable